helm/soperator-activechecks/scripts/all-reduce-perf-nccl-in-docker.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#SBATCH --deadline="now+4hours"
+#SBATCH --time=10:00
+#SBATCH --gpus-per-node=8
+#SBATCH --exclusive
+#SBATCH --mem=0
+
+
+echo "Running all_reduce_perf_nccl_in_docker check on $(hostname)..."
+
+mkdir -p /tmp/soperatorchecks/a
+mkdir -p /tmp/soperatorchecks/b
+
+srun --gpus=8 docker run --rm \
+  --gpus=all --device=/dev/infiniband \
+  -v /tmp/soperatorchecks/a:/a \
+  --mount type=bind,source=/tmp/soperatorchecks/b,target=/b \
+  {{ include "activecheck.image.docker" . }} \
+  bash -c "NCCL_P2P_DISABLE=1 NCCL_SHM_DISABLE=1 NCCL_ALGO=Ring all_reduce_perf -b 512M -e 8G -f 2 -g 8"
```

Comment:
We need to think how parametrize that for different GPU types. For example GBx00 has 4 GPU

Reply:
I'd do this separately, that's not the only issue with GBx00 cards